root/static/scripts/common/components/TagEditor.js

```diff
@@ -575,9 +619,10 @@ function createInitialTagState(
 }
 
 function init_tag_editor(Component, mountPoint) {
-  return function (entity, aggregatedTags, userTags, more) {
+  return function ($c, entity, aggregatedTags, userTags, more) {
```

Comment:
Ah, the call to `MB.init_main_tag_editor` in root/entity/tags.tt will need to be passed a `$c` now though. For now, we can probably do this:

```
          MB.init_main_tag_editor(
            {user_exists: true},
            {entityType: '[% entity_type %]', gid: '[% entity.gid %]'},
            [% closing_tag_escape(tags_json) %],
            [% closing_tag_escape(user_tags_json) %],
            [% more_tags ? 'true' : 'false' %]
          );
```
i.e. pass `{user_exists: true}` for `$c`. It's not a real context, but should work fine because `user_exists` is the only property accessed by the component (and JS inside TT is not checked by Flow). We'll be removing `init_tag_editor` soon once all entities are converted, anyway.